op-chain-ops/genesis/db_migration.go

```diff
@@ -147,7 +196,7 @@ func CheckWithdrawals(db vm.StateDB, withdrawals []*crossdomain.LegacyWithdrawal
 	for slot := range knownSlots {
 		_, ok := slots[slot]
 		if !ok {
-			return fmt.Errorf("Unknown input message: %s", slot)
+			//return nil, fmt.Errorf("Unknown input message: %s", slot)
```

Comment:
This was deliberately commented out for similar reasons re: our available data as stated above.